packages/react-components/src/organisms/__tests__/TeamCreateOutputForm.test.tsx

```diff
@@ -57,184 +50,242 @@ it('renders the form', async () => {
+    });
+    fireEvent.change(screen.getByLabelText(/title/i), {
+      target: { value: data.title },
+    });
+    fireEvent.change(screen.getByLabelText(/description/i), {
+      target: { value: data.description },
+    });
+    userEvent.type(screen.getByLabelText(/Select the option/i), data.type);
+    fireEvent.keyDown(screen.getByLabelText(/Select the option/i), {
+      keyCode: ENTER_KEYCODE,
+    });
+  };
+  const submitForm = async () => {
+    const button = screen.getByRole('button', { name: /Share/i });
+    userEvent.click(button);
 
-  userEvent.type(screen.getByLabelText(/date published/i), '2022-03-24');
+    await waitFor(() => {
+      expect(screen.getByRole('button', { name: /Share/i })).toBeEnabled();
+      expect(screen.getByRole('button', { name: /Cancel/i })).toBeEnabled();
+    });
+  };
 
-  userEvent.click(screen.getByLabelText(/Labs/i));
-  await waitFor(() =>
-    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
-  );
-  userEvent.click(screen.getByText('One Lab'));
+  it('can submit a form with minimum data', async () => {
+    setupForm();
+    await submitForm();
+    expect(saveFn).toHaveBeenLastCalledWith(expectedRequest);
+  });
 
-  userEvent.click(screen.getByLabelText(/Authors/i));
-  await waitFor(() =>
-    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
-  );
-  userEvent.click(screen.getByText(/Chris Reed/i));
+  it('can submit a lab', async () => {
+    getLabSuggestions.mockResolvedValue([
+      { label: 'One Lab', value: '1' },
+      { label: 'Two Lab', value: '2' },
+    ]);
+    setupForm();
+    userEvent.click(screen.getByLabelText(/Labs/i));
+    await waitFor(() =>
+      expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
+    );
+    userEvent.click(screen.getByText('One Lab'));
 
-  userEvent.click(screen.getByLabelText(/Authors/i));
-  await waitFor(() =>
-    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
-  );
-  userEvent.click(screen.getByText('Chris Blue'));
+    await submitForm();
+    expect(saveFn).toHaveBeenLastCalledWith({
+      ...expectedRequest,
+      labs: ['1'],
+    });
+  });
+  it('can submit existing internal and external and create a new external author', async () => {
+    getAuthorSuggestions.mockResolvedValue([
+      {
+        user: { ...createUserResponse(), displayName: 'Chris Blue' },
+        label: 'Chris Blue',
+        value: 'u2',
+      },
+      {
+        user: {
+          ...createUserResponse(),
+          email: undefined,
+          displayName: 'Chris Reed',
+        },
+        label: 'Chris Reed (Non CRN)',
+        value: 'u1',
+      },
+    ]);
+    setupForm();
 
-  userEvent.click(screen.getByLabelText(/Authors/i));
-  await waitFor(() =>
-    expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
-  );
+    userEvent.click(screen.getByLabelText(/Authors/i));
+    await waitFor(() =>
+      expect(screen.queryByText(/loading/i)).not.toBeInTheDocument(),
+    );
```

Comment:
This is repeated, should it be moved to a helper function?

Reply:
Its only repeated twice. The third time you have to type in an external author in order to create the new external author